test/component/cypress/specs/components/Pagination.cy.jsx

```diff
@@ -173,15 +173,15 @@ describe('Pagination', () => {
       cy.window().its('scrollY').should('equal', 0)
     })
   })
-  context('when you navigate to page 2 by clicking the Next link', () => {
+  context('when you navigate to page 2 by clicking the Next page link', () => {
```

Comment:
```suggestion
  context('when you navigate to page 2 by clicking the next page link', () => {
```

my bad for capitalising that yesterday

quick question: do we not have a similar but reverse test case for `when you navigate from page 2 by clicking the previous page link`?

Reply:
https://github.com/uktrade/data-hub-frontend/blob/main/test/component/cypress/specs/Resource/Paginated.cy.jsx#L55